src/components/EventCard/index.js

```diff
@@ -90,7 +91,7 @@ export default class EventCard extends PureComponent {
               {isPending && <EventWarning id="str.pendingConfirmation" message="Pending Confirmation" />}
               {isUpcoming && <EventWarning id="str.upcoming" message="Upcoming" type="upcoming" />}
               <Typography variant="headline" className={classes.eventCardName}>
-                {name}
+                {name}<FavoriteButton event={this.props.event} />
```

Comment:
move `FavoriteButton` out of `Typography` and wrap both in a `span` or `div`. yes it does work this way, but its not proper since its not meant to be part of the Typography.